RHEL/6/profiles/stig-rhel6-disa.xml

```diff
@@ -1,22 +1,14 @@
 <Profile id="stig-rhel6-server-upstream" extends="common">
```

Comment:
@shawndwells @redhatrises Should this Profile still be called `stig-rhel6-server-upstream`?

Reply:
@yuumasato that's a bug. Profile should be stig-rhel6-disa